### DISCUSSION TITLE
Pruning of Old kernel Images

### DISCUSSION THREAD

**Discussion by jayasimha-raghavan-unskript** [jupyter/enterprise_gateway#1337] [Q&A]
Hello @kevin-bates  

We have a scenario wherein we have `enterprise_gateway`  deployed as a POD on a Kubernetes Cluster. 
The KernelSpec policy is set to Download `Always`. The issue is that the Older images that are pulled by 
the KIP is not cleaned up, as a result the Node has these disk images built up and i subsequently disk pressure
from the Node. Is there a way in KIP that can cleanup older pulled images periodically? 

Thank you!

**Comment by kevin-bates** [comment]
This is a good idea!  This capability doesn't exist but such an enhancement would be awesome.  We could then rename KIP to KIM (Kernel Image Manager).  :smile:  (That was a joke, name changes are expensive.)
